cdc/sink/codec/java.go

```diff
@@ -63,13 +64,12 @@ const (
 	// JavaSQLTypeTIMESTAMP_WITH_TIMEZONE JavaSQLType = 2014
 )
 
-// MysqlToJavaType converts the mysql protocol types to java sql types
-func MysqlToJavaType(mysqlType byte) JavaSQLType {
+// MySQLType2JavaType converts the mysql protocol types to java sql types
+// take https://github.com/alibaba/canal/blob/master/dbsync/src/main/java/com/taobao/tddl/dbsync/binlog/event/RowsLogBuffer.java#L132 as reference
```

Comment:
```suggestion
// take https://github.com/alibaba/canal/blob/b54bea5e3/dbsync/src/main/java/com/taobao/tddl/dbsync/binlog/event/RowsLogBuffer.java#L1 as reference
```

Please use permalink.